rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -42,94 +42,136 @@
 log = RotkehlchenLogsAdapter(logger)
 
 SMALLEST_AIRDROP_SIZE = 20900
-AIRDROPS = {
-    'uniswap': (
+
+
+class Airdrop(NamedTuple):
+    url: str
+    asset: Asset
+    identifier: str
+    name: str
+    icon: str
+
+
+AIRDROPS: dict[str, Airdrop] = {
+    'uniswap': Airdrop(
         # is checksummed
         'https://gist.githubusercontent.com/LefterisJP/d883cb7187a7c4fcf98c7a62f45568e7/raw/3718c95d572a29b9c3906d7c64726d3bd7524bfd/uniswap.csv',  # noqa: E501
         A_UNI,
         'https://app.uniswap.org/',
+        'Uniswap',
+        'uniswap.svg',
```

Comment:
can you use kwargs when creating the objects? We tend to do it to avoid errors and have better type checking later

```suggestion
    'uniswap': Airdrop(
        # is checksummed
        url='https://gist.githubusercontent.com/LefterisJP/d883cb7187a7c4fcf98c7a62f45568e7/raw/3718c95d572a29b9c3906d7c64726d3bd7524bfd/uniswap.csv',  # noqa: E501
        asset=A_UNI,
        identifier='https://app.uniswap.org/',
        name='Uniswap',
        icon='uniswap.svg',
```